### PR TITLE
add url parameter for assistant to docs

### PR DESCRIPTION
**Pull request by thibaultmol**
I also would have not known it was a thing if I hadn't seen it on discord. but it's not actually mentioned in the docs yet

https://kagifeedback.org/d/5254-hyperlink-directly-to-specific-assistants

### CODE REVIEW

**Comment by thibaultmol** [comment]
PLEASE CHECK THE ASSISTANT LIST before merging. There are mistakes in there, I just don't know what the right ones are atm

**Comment by david-ros** [comment]
@thibaultmol I think you didn't mean to include this in your commit:

For this addition, a natural location would be right after the **LLMs Available in The Assistant** section, adding a clear explanation on the use of the `profile` parameter to select a specific LLM, like `gpt-4o` or `claude3.5`. Here's the updated content for that section:

**Comment by thibaultmol** [comment]
yep, had also noticed it right before you mentioned it xd
not sure why it didn't build before, hoping it builds now.

But yeah, someone should still check and change the exact profile names because some of them are def not correct

File: docs/kagi/ai/custom-assistants.md [hunk 1]
```diff
@@ -47,6 +47,18 @@ Remember that well-crafted instructions lead to more accurate and useful respons
 
 Select your Custom Assistant from the model-selection dropdown menu below the prompt bar to begin interaction.
 
+## URL Parameter for Custom Assistants
+
+You can specify a custom assistant by adding a `profile` parameter to the URL, using either the name (in lowercase, hyphenated format) or the UUID of your custom assistant.
```
Comment (nobodywasishere) [inline docs/kagi/ai/custom-assistants.md:52]: It's just lowercase format (after next release)

File: docs/kagi/ai/assistant.md [hunk 2]
```diff
@@ -166,6 +166,12 @@ You can quickly access Assistant from Kagi Search by using the following [bangs]
 
 You can also access the **Code** Custom Assistant with the `!code` bang.
 
+## URL Parameter for Model Selection
+
+You can specify a particular model in the Assistant's URL by including a `profile` parameter.
+`https://kagi.com/assistant?profile=gpt_4o&q=%s`
+The available model names include: `gpt_4o` `gpt_4o_mini` `gpt_4` `gpt_4_turbo` `claude_3_haiku` `claude_3_opus` `claude_3_sonnet` `code` `gemini_pro` `llama-3-405b` `mistral_pixtral` `mistral_large`
```
Comment (nobodywasishere) [inline docs/kagi/ai/assistant.md:173]: `code` is one of the default assistants, not a model, though that's semantics for this.

**Comment by thibaultmol** [comment]
Feel free to edit whatever and then merge

**Comment by nobodywasishere** [comment]
I've updated the profile names and added more docs on other parameters. I'll leave it to @david-ros to approve / merge.

**Comment by thibaultmol** [comment]
Probably a dumb thing: i put gpt4o first because that's the recommended model of all the openai's ones (people might instinctively pick the first one in the list even though that's not the best one. And it's the most expensive one(

**Pull request merged**